cmd/finch/main_native.go

```diff
@@ -41,7 +43,11 @@ func xmain(logger flog.Logger,
 		ecc,
 	)
 	if err != nil {
-		return fmt.Errorf("failed to load config: %w", err)
+		if errors.Is(err, os.ErrPermission) {
+			logger.Warnf("Failed to load config, using default values. Are you running as root? (%s)", err)
```

Comment:
Slight nitpick to make the message clear: Prefer "You may need to be root or use sudo"

Reply:
Updated message PTAL